Desktop/widgets/sourceitem.cpp

```diff
@@ -102,10 +96,10 @@ void SourceItem::_setUp()
 	}
 	else if (_nativeModel)
 	{
-		connect(_nativeModel, &QAbstractItemModel::dataChanged,			this,	&SourceItem::nativeModelChanged);
-		connect(_nativeModel, &QAbstractItemModel::rowsInserted,		this,	&SourceItem::nativeModelChanged);
-		connect(_nativeModel, &QAbstractItemModel::rowsRemoved,			this,	&SourceItem::nativeModelChanged);
-		connect(_nativeModel, &QAbstractItemModel::modelReset,			this,	&SourceItem::nativeModelChanged);
+		connect(_nativeModel, &QAbstractItemModel::dataChanged,			this,	[this]() { _listControl->model()->sourceTermsChanged(); });
+		connect(_nativeModel, &QAbstractItemModel::rowsInserted,		this,	[this]() { _listControl->model()->sourceTermsChanged(); });
+		connect(_nativeModel, &QAbstractItemModel::rowsRemoved,			this,	[this]() { _listControl->model()->sourceTermsChanged(); });
```

Comment:
Ok, this is not necessarily a complaint, or request for change but I am not sure why you did:
`connect(_nativeModel, &QAbstractItemModel::dataChanged,			this,	[this]() { _listControl->model()->sourceTermsChanged(); });`
When you could also do:
`connect(_nativeModel, &QAbstractItemModel::dataChanged,			this,	_listControl->model(), &ListModel::sourceTermsChanged());` 
(That is, if `_listControl->model()` is a ListModel.